### DISCUSSION TITLE
Can't Mount NAS - What am I doing wrong?

### DISCUSSION THREAD

**Discussion by MDEsswein64** [rern/rAudio#723] [Bug Reports]
My Runeaudio box died the other day, so I decided to upgrade to rAudio-1.  I got it set up but I can't figure out how to mount my NAS music storage. any help is greatly appreciated.

I'm running: 20211222 on a Raspberry Pi 3 Model B Rev 1.2 with a HiFiBerry DAC+

Here is what shows in the /Settings/System/Storage...

# cat /etc/fstab
PARTUUID=c10a8be4-01  /boot  vfat  defaults,noatime  0  0
PARTUUID=c10a8be4-02  /      ext4  defaults,noatime  0  0
//192.168.86.65/HomeBackUpServer/MDEBackup/iTunes  /mnt/MPD/NAS/NASMusic cifs defaults,noauto,bg,soft,timeo=30  0  0

# mount | grep ^/dev
/dev/mmcblk0p1 on /boot type vfat (rw,noatime,fmask=0022,dmask=0022,codepage=437,iocharset=ascii,shortname=mixed,errors=remount-ro)
/dev/mmcblk0p2 on / type ext4 (rw,noatime)

**Comment by MDEsswein64** [comment]
I figured it out.  I needed to use the vers=1.0 option.

**Comment by R101G** [comment]
Samba v3 seems unnecessary for a music server and Asus, for one, have said it needs too much computing power for them to include it in a firmware update. So it might eliminate some potential NAS problems if vers=2.0 was the default for rAudio. 

@rern?

**Reply by teslasparks301** [reply]
My NAS music share is mounted with vers 3.1.1 without any undue stress. rAudio-1 is running on a Raspberry Pi 3 Model B Rev 1.2. My points are 1) one should definitely not be using vers=1 anymore and 2) rpi has enough horsepower to use versions above 1. This is from the perspective of client and not server. No, v3 is not necessary for a music server but it is for a modern, secure SMB connection.

I've used samba for 20 years and it has caused me untold fits to this very day so I completely understand resorting to vers=1 but it really means that the samba dox need to be revisited. Some setting(s) need to be adjusted. Also any hardware that can not serve at vers > 1 is probably quite old and should be retired.

My take on the subject, FWIW.

**Reply by R101G** [reply]
@teslasparks301 There is no problem with rAudio using v3, but some current routers and older NAS can't run v3. As there is no indication of why the mount fails, it might save new users some grief if vers=2.0 appeared as the default in the settings field. Anyone who wanted v3 could delete it. Apparently there is a large increase in the resources needed to run v3 rather than v2, which is why some equipment can't be upgraded.
